### DISCUSSION TITLE
WYSIWYG Drag & Drop editor like WIX

### DISCUSSION THREAD

**Discussion by davidjeba** [elementor/elementor#24778] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Block based page design is very difficult for those who want to migrate from Wix to Wordpress.
Wix frontend editor is 100000x more fluid than Elementor.

### Describe the solution you'd like.

If Elementor can create a good alternate to Wix editor or even better than wix
Elementor should use Figma as frontend editor.

### Describe alternatives you've considered.

Most of the web design starts from Figma, Adobe XD
if Either Figma or Adobe XD launches a plugin for Wordpress or any CMS which can act as a drag & drop editor
Elementor will become outdated

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
> if Either Figma or Adobe XD launches a plugin for Wordpress or any CMS which can act as a drag & drop editor
Elementor will become outdated

Did you search for it yet? There is https://www.figma.com/community/plugin/1169206626650589972 or https://www.figma.com/community/plugin/1265873702834050352 already. Why would Elementor be outdated? Elementor is a WordPress builder, Figma is a design tool. Yes, some people use Figma + Elementor but many more don't. So they serve a different purpose with some overlapping groups. 

I receive designs from Figma but I create the Elementor themes from scratch and would never use an exporter to do it because they won't create the needed DOM structure.
